src/install/haproxy/whatsNext.mdx

```diff
@@ -0,0 +1,1766 @@
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **USERNAME**
+      </td>
+
+      <td>
+        Username for the HAProxy connection.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **PASSWORD**
+      </td>
+
+      <td>
+        Password for the given user.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **HA_PROXY_CLUSTER_NAME**
+      </td>
+
+      <td>
+        A user-defined name to uniquely identify the cluster being monitored.
+        **Required**.
+      </td>
+
+      <td>
+        N/A
+      </td>
+
+      <td style={{ 'text-align': 'center' }}>
+        M/I
+      </td>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **METRICS**
+      </td>
+
+      <td>
+        Set to `true` to enable Metrics-only collection.
+      </td>
+
+      <td>
+        false
+      </td>
+
+      <td style={{ 'text-align': 'center' }}/>
+    </tr>
+
+    {
+      ' '
+    }
+
+    <tr>
+      <td>
+        **INVENTORY**
+      </td>
+
+      <td>
+        Set to `true` to enable Inventory-only collection.
+      </td>
+
+      <td>
+        false
+      </td>
+
+      <td style={{ 'text-align': 'center' }}/>
+    </tr>
+  </tbody>
+</table>
+
+The values for these settings can be defined in several ways:
+
+* Adding the value directly in the config file (the most common way).
+* Replacing the values from environment variables using the `{{}}` notation. This requires infrastructure agent version 1.14.0 or higher. For more information, see [Infrastructure configuration](/docs/infrastructure/install-infrastructure-agent/configuration/configure-infrastructure-agent/#passthrough) or see [the environment variable example](#example5).
+* Using secrets management. Use this to protect sensitive information, such as passwords that would be exposed in plain text in the configuration file. For more information, see [Secrets management](/docs/integrations/host-integrations/installation/secrets-management).
+
+## Labels  [#labels]
+
+You can further decorate your metrics using labels. Labels allow you to add attributes (key/value pairs) to your metrics, which you can then use to query, filter, or group your metrics.
+
+Our default sample config file includes examples of labels but, because they're not mandatory, you can remove, modify, or add new ones of your choice.
+
+```
+ labels:
+   env: production
+   role: load_balancer
+```
+
+
+## Metric data [#metrics]
+
+The HAProxy integration collects the following metric data attributes. Each metric name is prefixed with a category indicator and a period, such as `backend.`, `frontend.`, or `server.`.
+
+### HAProxy backend sample metrics [#backend-sample]
```

Comment:
these tables are a bit long. should we consider putting them in collapsers organized by your current h3 titles? (i'm thinking about that long doc you edited recently where we ended up needing to do that anyway)